docs/tutorials/tutorial-sketches-theta.md

```diff
@@ -0,0 +1,302 @@
+
+![Load data view with pasted data](../assets/tutorial-theta-01.png)
+
+Leave the source type as `inline` and click **Apply** and **Next: Parse data**.
+Parse the data as CSV, with included headers:
+
+![Parse raw data](../assets/tutorial-theta-02.png)
+
+Accept the default values in the **Parse time**, **Transform**, and **Filter** stages.
+
+In the **Configure schema** stage, enable rollup and confirm your choice in the dialog. Then set the query granularity to `day`.
+
+![Configure schema for rollup and query granularity](../assets/tutorial-theta-03.png)
+
+You also add the Theta sketch during this stage. Select **Add metric**.
+Define the new metric as a Theta sketch with the following details:
+* **Name**: `theta_uid`
+* **Type**: `thetaSketch`
+* **Field name**: `uid`
+* **Size**: Leave at the default value, `16384`.
+* **Is input theta sketch**: Leave at the default value, `False`.
+
+![Create Theta sketch metric](../assets/tutorial-theta-04.png)
+
+Click **Apply** to add the new metric to the data model.
+
+
+We have to perform one more step to complete the data model. We are not interested in individual user ID's, only the unique counts. Right now, `uid` is still in the data model. Let's get rid of that!
+
+Click on the `uid` column in the data model and delete it using the trashcan icon on the right:
+
+![Delete uid column](../assets/tutorial-theta-05.png)
+
+For the rest of the **Load data** wizard, set the following options:
+* **Partition** stage: Set **Segment granularity** to `day`.
+* **Tune**: Leave the default options.
+* **Publish**: Set the datasource name to `ts_tutorial`.
+
+On the **Edit spec** page, your final input spec should look like the following JSON: 
+
+```json
+{
+  "type": "index_parallel",
+  "spec": {
+    "ioConfig": {
+      "type": "index_parallel",
+      "inputSource": {
+        "type": "inline",
+        "data": "date,uid,show,episode\n2022-05-19,alice,Game of Thrones,S1E1\n2022-05-19,alice,Game of Thrones,S1E2\n2022-05-19,alice,Game of Thrones,S1E1\n2022-05-19,bob,Bridgerton,S1E1\n2022-05-20,alice,Game of Thrones,S1E1\n2022-05-20,carol,Bridgerton,S1E2\n2022-05-20,dan,Bridgerton,S1E1\n2022-05-21,alice,Game of Thrones,S1E1\n2022-05-21,carol,Bridgerton,S1E1\n2022-05-21,erin,Game of Thrones,S1E1\n2022-05-21,alice,Bridgerton,S1E1\n2022-05-22,bob,Game of Thrones,S1E1\n2022-05-22,bob,Bridgerton,S1E1\n2022-05-22,carol,Bridgerton,S1E2\n2022-05-22,bob,Bridgerton,S1E1\n2022-05-22,erin,Game of Thrones,S1E1\n2022-05-22,erin,Bridgerton,S1E2\n2022-05-23,erin,Game of Thrones,S1E1\n2022-05-23,alice,Game of Thrones,S1E1"
+      },
+      "inputFormat": {
+        "type": "csv",
+        "findColumnsFromHeader": true
+      }
+    },
+    "tuningConfig": {
+      "type": "index_parallel",
+      "partitionsSpec": {
+        "type": "hashed"
+      },
+      "forceGuaranteedRollup": true
+    },
+    "dataSchema": {
+      "dataSource": "inline_data",
```

Comment:
should this be `ts_tutorial` to match line 187?

Reply:
I didn't see anywhere in the Load data UI that you could set the datasource name for inline data, so there many need to be an instruction to set the `datasource` name or potentially change the ds name in the tutorial to match this.

Reply:
described in line 131